scss/civicrm/administer/events/_badgelayout.scss

```diff
@@ -18,6 +18,11 @@
           min-width: 100px !important;
         }
       }
+
+      .six {
```

Comment:
what's this class?

Reply:
This class is applied by Vanilla civicrm when they want input to be `width: 6em` for us, I made it `width: 100px`